source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -101,7 +101,7 @@ Terms
 Server
 ``````
 
-A mongod or mongos process.
+A mongod or mongos process, or a load balancer.
```

Comment:
I think we should have load balanced mode tests as well and likely add something to this section of the load balancer spec: https://github.com/mongodb/specifications/blob/master/source/load-balancers/load-balancers.rst#monitoring

Reply:
Added test and short description to load-balancer spec.